data/teams_sample.csv

```diff
@@ -0,0 +1,4 @@
+team_id,franchiseId,teamName,abbreviation,Stadium,link
+1,23,Atlanta United,ATL,Mercedes-Benz Stadium,/api/v1/teams/1
+4,16,Chicago Fire,CHI,SeatGeek Stadium,/api/v1/teams/4
+26,14,FC Cincinnati,CIN,Nippert Stadium,/api/v1/teams/26
```

Comment:
Might need to add more data later on to cover tests for certain statistic calculations; but we can add specific data as needed.

Reply:
Agreed. Gtg for now.